.github/workflows/prod-build.yml

```
@@ -167,8 +167,8 @@ jobs:
           REACT_APP_MDN_PLUS_10Y_PLAN: price_1KeG02JNcmPzuWtRlrSiLTI6
 
           # Surveys.
-          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 1675252800000 # (new Date("2023-02-01 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_END_FIREFOX_WEB_COMPAT_2023: 1675684800000 # (new Date("2023-02-06 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 1675771200000 # (new Date("2023-02-07 12:00:00Z")).getTime()
```

No need to change the start date:

```suggestion
          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 1675252800000 # (new Date("2023-02-01 12:00:00Z")).getTime()
```